### PR TITLE
contrib/tig: new package (2.5.8)

### PR DESCRIPTION
**Pull request by wezm**

### CODE REVIEW

File: contrib/tig/template.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+hostmakedepends = ["gmake", "asciidoc", "xmlto", "pkgconf"]
+makedepends = ["ncurses-devel"]
+depends = ["git"]
+pkgdesc = "Text-mode interface for git"
+maintainer = "Wesley Moore <wes@wezm.net>"
+license = "GPL-2.0-or-later"
+url = "https://github.com/jonas/tig"
+source = f"{url}/releases/download/{pkgname}-{pkgver}/{pkgname}-{pkgver}.tar.gz"
+sha256 = "b70e0a42aed74a4a3990ccfe35262305917175e3164330c0889bd70580406391"
+
```
Comment (q66) [inline contrib/tig/template.py:18]: drop extra whitespace

File: contrib/tig/template.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+# test suite tries to access /dev/tty which fails
+options = ["!check"]
+
+
+def post_install(self):
+    self.install_file(
```
Comment (q66) [inline contrib/tig/template.py:24]: use install_completion

File: contrib/tig/template.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+pkgname = "tig"
+pkgver = "2.5.8"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_gen = []
```
Comment (q66) [inline contrib/tig/template.py:5]: any reason the autotools cannot be regenerated?

**Review by q66 — CHANGES_REQUESTED** [review]

File: contrib/tig/template.py (hunk 3 above)
Comment (wezm) [reply contrib/tig/template.py:5]: No reason, configure was already present so was making use of it but if the done thing is to regenerate it then I'm happy to make that change.

**Comment by wezm** [comment]
I addressed the review comments

**Pull request merged**